pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1145,6 +1160,79 @@ SELECT description
 		},
 	),
 
+	"pg_relation_is_updatable": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types:      tree.ArgTypes{{"reloid", types.Oid}, {"include_triggers", types.Bool}},
+			ReturnType: tree.FixedReturnType(types.Int4),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				oidArg := tree.MustBeDOid(args[0])
+				oid := int(oidArg.DInt)
+				table, err := ctx.Planner.GetImmutableTableInterfaceByID(ctx.Ctx(), oid)
+				if err != nil {
+					//nolint:returnerrcheck
+					return nonUpdatableEvents, nil
+				}
+				tableDesc, ok := table.(catalog.TableDescriptor)
+				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
+					return nonUpdatableEvents, nil
+				}
+
+				// pg_relation_is_updatable was created for compatibility. This
+				// should return the update events the relation supports, but as crdb
+				// does not support updatable views or foreign tables, right now this
+				// basically return allEvents or none.
+				return allUpdatableEvents, nil
+			},
+			Info:       `Returns the update events the relation supports.`,
+			Volatility: tree.VolatilityStable,
+		},
+	),
+
+	"pg_column_is_updatable": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"reloid", types.Oid},
+				{"attnum", types.Int2},
+				{"include_triggers", types.Bool},
+			},
+			ReturnType: tree.FixedReturnType(types.Bool),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				oidArg := tree.MustBeDOid(args[0])
+				attNumArg := tree.MustBeDInt(args[1])
+				oid := int(oidArg.DInt)
+				attNum := uint32(attNumArg)
+				if attNumArg < 0 {
+					// system columns are not updatable
+					return tree.DBoolFalse, nil
+				}
+				table, err := ctx.Planner.GetImmutableTableInterfaceByID(ctx.Ctx(), oid)
+				if err != nil {
+					//nolint:returnerrcheck
+					return tree.DBoolFalse, nil
+				}
+				tableDesc, ok := table.(catalog.TableDescriptor)
+				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
+					return tree.DBoolFalse, nil
+				}
+
+				column, err := tableDesc.FindColumnWithID(descpb.ColumnID(attNum))
+				if err != nil || column.IsComputed() {
+					//nolint:returnerrcheck
```

Comment:
Hi Oliver, let me check as I saw this on postgres code:
https://github.com/postgres/postgres/blob/master/src/backend/rewrite/rewriteHandler.c#L2754

`rel = try_relation_open(reloid, AccessShareLock);

	/*
	 * If the relation doesn't exist, return zero rather than throwing an
	 * error.  This is helpful since scanning an information_schema view under
	 * MVCC rules can result in referencing rels that have actually been
	 * deleted already.
	 */
	if (rel == NULL)
		return 0;`

Reply:
Look at this:

miguelnovelo=# SELECT pg_relation_is_updatable(1, true)
;
 pg_relation_is_updatable 
--------------------------
                        0
(1 row)

miguelnovelo=# SELECT count(*) FROM pg_class WHERE oid = 1;
 count 
-------
     0
(1 row)

miguelnovelo=#

Reply:
```
otan=# \d bobby
                       Table "public.bobby"
 Column |          Type           | Collation | Nullable | Default
--------+-------------------------+-----------+----------+---------
 a      | geometry(Geometry,4326) |           |          |

otan=# select pg_column_is_updatable('bobby'::regclass::oid,55::int2,false);
 pg_column_is_updatable
------------------------
 t
(1 row)
```

if the column doesn't exist (55 certainly doesn't), looks like it returns true?